### PR TITLE
completed entire section by adding on the title and 'view more' button. Also fixed the padding issue between svg and description on smaller screens

### PR DESCRIPTION
**Pull request by jilly477**

### CODE REVIEW

**Comment by jilly477** [comment]
> _No description provided._

completed entire section by adding on the title and 'view more' button. Also fixed the padding issue between svg and description on smaller screens

File: sections/home/ImpactSection.jsx [hunk 1]
```diff
@@ -3,32 +3,59 @@ import styled from "styled-components";
 import ImpactGraphic1 from "../../public/home/impact-section-graphic-1.svg";
 import ImpactGraphic2 from "../../public/home/impact-section-graphic-2.svg";
 import ImpactGraphic3 from "../../public/home/impact-section-graphic-3.svg";
-import { ImpactCard } from "../../components";
+import { ImpactCard, Text, Button } from "../../components";
 import { media } from "../../utils";
+import { baseTheme } from "../../theme";
+import Link from "next/link";
```
Comment (xsharonhe) [inline sections/home/ImpactSection.jsx]: This should be at the top of the file. It always goes importing external packages first and then own files

**Review by xsharonhe — COMMENTED** [review]
Small change and then it's good.

File: sections/home/ImpactSection.jsx (hunk 1 above)
Comment (jilly477) [reply sections/home/ImpactSection.jsx]: baseTheme as well? I saw baseTheme near the end of the WhatisHappening section and now it looks like this: 
image
Is this good?

Comment (xsharonhe) [reply sections/home/ImpactSection.jsx]: yes that's fine. baseTheme is our own files so it does go at the end.

**Pull request merged**